NEWS

```diff
@@ -7,6 +7,8 @@
 
   Major changes between OpenSSL 1.1.1 and OpenSSL 3.0.0 [under development]
 
+      o X509 certificates signed using SHA1 are no longer allowed at security
+        level 1
```

Comment:
1 or higher.

Reply:
Perhaps again mention that the default security level is 1 for TLS, which means that certificates signed with SHA-1 are by default no longer trusted to authenticate TLS servers or clients.